fuse_models/src/odometry_2d_publisher.cpp

```diff
@@ -104,7 +96,7 @@ void Odometry2DPublisher::notifyCallback(
 {
   // Find the most recent common timestamp
   const auto latest_stamp = synchronizer_.findLatestCommonStamp(*transaction, *graph);
-  if (latest_stamp == Synchronizer::TIME_ZERO)
+  if (latest_stamp == Synchronizer::ROS_TIME_ZERO)
```

Comment:
If `rclcpp::Time` had an `is_valid()` method it would be convenient here `if (latest_stamp.is_valid()) ...` so that `Synchronizer::ROS_TIME_ZERO` wasn't necessary.

Reply:
Wouldn't it be more explicit that it's a comparison to Time(0, 0) here so readers of the code know that it's something like what TF2 is doing?

Reply:
okay https://github.com/locusrobotics/fuse/pull/283/commits/bdedc95997d9e5fe34614bdeace81f21fabb654f